certora/harnesses/RewardsManagerHarness.sol

```diff
@@ -77,6 +77,35 @@ contract RewardsManagerHarness is RewardsManager {
     }
 
     // space to create your own destiny 
+    function getAddress() public view returns(address) {
```

Comment:
Hi, could I ask you to make several changes, please, and resubmit your merge request?
Could you rename this harness a little bit and fix scripts according to it. We need it because you added your own methods in harness and if we will merge all changed harnesses together we will have a lot of issue with running a verification.

Reply:
I changed the name, should I cancel the PR and make a new one or is this fine?